src/test/java/operationsTest.java

```diff
@@ -1,11 +1,41 @@
+import exceptions.*;
+import org.json.*;
 import org.junit.jupiter.api.Test;
-
+import java.io.IOException;
+import java.nio.file.Paths;
 import static org.junit.jupiter.api.Assertions.*;
+import static org.mockito.Mockito.*;
 
 class operationsTest {
+
```

Comment:
Where are the test cases for set,  get, and  delete

Reply:
In this pr i just implemented the create